src/main/java/tds/content/controllers/ContentController.java

```diff
@@ -79,4 +81,24 @@ public ContentController(final ContentService contentService) {
 //            .contentLength(resource.contentLength())
```

Comment:
This is actually something I missed with some of my work, but would you mind deleting this comment out code before merging?